components/dashboard/src/projects/Prebuilds.tsx

```diff
@@ -170,7 +188,7 @@ export default function () {
                 <div className="py-3 pl-3">
                     <DropDown prefix="Prebuild Status: " contextMenuWidth="w-32" entries={statusFilterEntries()} />
                 </div>
-                {(!isLoadingPrebuilds && prebuilds.length === 0) &&
+                {(!isLoadingPrebuilds && prebuilds.length === 0 && !props.isAdminDashboard) &&
```

Comment:
**praise**: Nice catch! I was just about to check this out. ⚾

Reply:
Ended up removing this check for #5022.